index.html

```diff
@@ -54,15 +54,15 @@ <h1>InCov</h1>
         1. <div id="1"></div>
         definisi COVID-19
         -->
-        <div id="top-wrapper" class="top-wrapper">
+        <div id="beranda" class="top-wrapper">
             <div class="container">
                 <div class="top-wrapper-left">
                     <h2>APA ITU <span>COVID-19</span> ?</h2>
```

Comment:
change media query at max width 670px of top-wrapper h2 font-size from 25px to 40px